kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -173,10 +171,30 @@
   public static final Boolean INCLUDE_KAFKA_DATA_DEFAULT =                 false;
   public static final ConfigDef.Importance INCLUDE_KAFKA_DATA_IMPORTANCE =
       ConfigDef.Importance.LOW;
-  public static final String INSTANCE_KAFKA_DATA_DOC =
+  public static final String INCLUDE_KAFKA_DATA_DOC =
       "Whether to include an extra block containing the Kafka source topic, offset, "
       + "and partition information in the resulting BigQuery rows.";
 
+  public static final String INCLUDE_KAFKA_KEY_CONFIG =                   "includeKafkaKey";
```

Comment:
What do you think of getting rid of `includeKafkaKey` and `includeKafkaData`, and just going with.. if `kafkaKeyFieldName` is not set or null, then it's not included, and same for `kafkaDataFieldName`?

Reply:
That sounds like a better way, updated accordingly!